gamification-twitter-services/src/main/java/io/meeds/twitter/gamification/storage/TwitterConsumerStorage.java

```diff
@@ -83,6 +84,7 @@ public class TwitterConsumerStorage {
 
   private HttpClient         client;
 
+  @Cacheable(value = "twitterAccount", key = "{#twitterUsername, #bearerToken}")
```

Comment:
Should we add `@CacheEvict` on `public void clearCache` and when `bearerToken` changes using `@CacheEvict(..., allEntries = true)` ?